display/core.py

```diff
@@ -2,32 +2,33 @@
 import operator
 import math
 import os
-from heppy.papas.pfobjects import Cluster
 
 class Display(object):
-    
-    def __init__(self, views=None):
+
+    def __init__(self, views=None, pads=None):
         ViewPane.nviews = 0
         if not views:
             views = ['xy', 'yz', 'xz']
         self.views = dict()
+        self.pads = pads
         for view in views:
-            if view in ['xy', 'yz', 'xz']:
-                self.views[view] = ViewPane(view, view,
-                                            100, -4, 4, 100, -4, 4)
+            if view in ['yz', 'xz']:
+                self.views[view] = ViewPane(view, view, 100, -4, 4, 100, -4, 4, pads=pads)
+            if view in ['xy']:
+                self.views[view] = ViewPane(view, view, 100, -2.8, 2.8, 100, -2.8, 2.8, pads=pads)
             elif 'thetaphi' in view:
                 self.views[view] = ViewPane(view, view,
                                             100, -math.pi/2, math.pi/2,
                                             100, -math.pi, math.pi,
-                                            500, 1000)
+                                            500, 1000, pads=pads)
 
-    def register(self, obj, layer, clearable=True):
+    def register(self, obj, layer, clearable=True, sides=None):
```

Comment:
Clearable was you and I am not sure what it does either 
have documented sides.